packages/typescript-to-proptypes/.mocharc.js

```diff
@@ -1,3 +1,4 @@
 module.exports = {
+  extension: ['js', 'ts', 'tsx'],
```

Comment:
Unrelated to this PR. But results in test re-running in watchmode if the files change. Including tsx extension for easier code-search.